assets/src/edit-story/app/api/apiProvider.js

```diff
@@ -226,9 +226,15 @@ function APIProvider({ children }) {
     [link]
   );
 
-  const getAllUsers = useCallback(() => {
-    return apiFetch({ path: addQueryArgs(users, { per_page: '-1' }) });
-  }, [users]);
+  const getUsers = useCallback(
+    (search = null) => {
+      // @todo per_page 10 is for testing, change later.
+      return apiFetch({
+        path: addQueryArgs(users, { per_page: '10', who: 'authors', search }),
```

Comment:
I'm thinking of sorting this so that the author of the current user would always be the first, otherwise, the author might not be among the first X authors.

Reply:
> author of the current user

author of the current _story_?

---

Does the current author really need to be the _first_ in the list? It needs to be somewhere in the list sure, but as long as it's the selected/current option, the position shouldn't matter.

https://github.com/WordPress/gutenberg/blob/2fdde1c7c14820f45b043724a4f73d7abd499a49/packages/editor/src/components/post-author/index.js#L42-L63 might help as reference.

Reply:
> author of the current story?

Yes!

> Does the current author really need to be the first in the list?

No, doesn't need to be always the first, was just an idea -- we can add it _if_ it doesn't exist as in Gutenberg.